src/platforms/Farcaster/index.ts

```diff
@@ -0,0 +1,12 @@
+import FarcasterIcon from "platforms/components/FarcasterIcon"
+import { PlatformAsRewardRestrictions, RewardData } from "platforms/types"
+
+export default {
+  icon: FarcasterIcon,
+  imageUrl: "/platforms/poap.png",
```

Comment:
whoopsie, wen poap?

Reply:
Nice catch! Changed in https://github.com/guildxyz/guild.xyz/pull/1301/commits/d8fc1742ada209187fbfe9365ed29f4ca5815790 👍